### PR TITLE
release: v1.0.0a5

### PR DESCRIPTION
**Pull request by lnielsen**
Signed-off-by: Lars Holm Nielsen <lars.holm.nielsen@cern.ch>

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@
 Changes
 =======
 
-Version 1.0.0 (released 2016-01-26)
-------------------------------------------
+Version 1.0.0 (released 2016-02-05)
```
Comment (jirikuncar) [inline CHANGES.rst]: a5?

Comment (lnielsen) [inline CHANGES.rst]: See comment on invenio-pages

Comment (lnielsen) [inline CHANGES.rst]: Sorry, meant invenio-rest

**Comment by lnielsen** [comment]
inveniosoftware/invenio-rest#23 and #30

**Pull request merged**